integration-tests/tests/src/tests/alias.ts

```diff
@@ -51,9 +53,13 @@ function addTestObjects(realm) {
   realm.commitTransaction();
 }
 
-module.exports = {
-  testAliasInSchema() {
-    const Person = {
+describe("Realm Alias", () => {
+  beforeEach(() => {
+    Realm.clearTestState();
+  });
+
+  it("supports remapping property names using mapTo", () => {
```

Comment:
🔜 I think the intention of this test is testing the different data types? Not 100% sure though

```suggestion
  it("supports all data types", () => {
```

Reply:
guess there are quite a few left out then :D

Reply:
I changed it to "supports defining aliases for properties in a schema".